templates/openstackclient/bin/ctlplane-inventory.yaml

```diff
@@ -0,0 +1,7 @@
+overcloud:
```

Comment:
as said above would also be good to have the different host groups in the inventory to be able to subscribe them to different channels.